docs/articles/documentation/using-testcafe/common-concepts/reporters.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: docs
+title: Reporters
+permalink: /documentation/using-testcafe/common-concepts/reporters/
+---
+# Reporters
+
+Reporters are plugins used to output test run reports in a certain format.
+
+TestCafe ships with the following reporters:
+
+* [spec](https://github.com/DevExpress/testcafe-reporter-spec) - used by default
+* [list](https://github.com/DevExpress/testcafe-reporter-list)
+* [minimal](https://github.com/DevExpress/testcafe-reporter-minimal)
+* [xUnit](https://github.com/DevExpress/testcafe-reporter-xunit)
+* [JSON](https://github.com/DevExpress/testcafe-reporter-json)
+
+You can also create a [custom repoter](/testcafe/documentation/extending-testcafe/creating-custom-reporter/) that will fit your needs.
+
+You can select a required reporter when working both through the [command line](/testcafe/documentation/using-testcafe/command-line-interface/#r-name-reporter-name) and [programmatically](/testcafe/documentation/using-testcafe/programming-interface/Runner/#reporter).
```

Comment:
BTW, testcafe also mixins `this.chalk` (that can be used for ANSI-coloring of the text) (API - https://github.com/chalk/chalk) and `this.moment` (can be used to deal with dates and time) (API - http://momentjs.com/). Reporter can also have `noColors` property (e.g. https://github.com/DevExpress/testcafe-reporter-json/blob/master/src/index.js#L3) which indicates that `chalk` and any coloring should be disabled for this reporter. This is useful for the machine-targeted formats, e.g. JSON or xUnit